### PR TITLE
Setup basic app

### PR DESCRIPTION
**Pull request by Rodgers-M**
#### What does this PR do?
sets up a basic application and server
Also sets up `eslint` 

#### How should this be manually tested?
Pull these changes, run `yarn install` then `yarn start` or `yarn start:dev`
Then head over to the index route. `GET`  `localhost:5000` 
You should get a welcome message.

### CODE REVIEW

**Review by danielotieno — COMMENTED** [review]
Can we work with Import/Export for ES6+ insted of require? 

I think it will be a good practice and also it will make code easy to read

**Comment by Rodgers-M** [comment]
This is still experimental in node currently

**Comment by danielotieno** [comment]
@Rodgers-M Yes, That's why we will use a package called [esm](https://www.npmjs.com/package/esm)

Steps:
`yarn add esm`

Then in `package.json` add script like this 

`"scripts": {
    "start": "node -r index.js",
    "start:dev": "nodemon -r index.js"
  },`

**Comment by Rodgers-M** [comment]
Am not a fan of experimental features as much but lets first hear what others have to say about it.

**Comment by danielotieno** [comment]
cool

**Comment by dessHub** [comment]
The PR looks good. As @danielotieno suggested the use of `import` is also my preference. I have been using it and I haven't experienced any issues so far.

**Comment by dessHub** [comment]
sorry, closed the PR instead of `comment`

**Review by dessHub — DISMISSED** [review]

File: app/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+import morgan from 'morgan'
+import routes from './routes'
+
+const app = express()
+
+app.use(bodyParser.json())
```
Comment (danielotieno) [inline app/index.js]: I can see we are using express 4.17 hence body parser has been re-added under the methods `express.json()` and `express.urlencoded()`

**Review by danielotieno — COMMENTED** [review]
I can see we are using express 4.17 hence body parser has been re-added under the methods `express.json()` and `express.urlencoded()` that means we don't need body-parser package

**Review by danielotieno — APPROVED** [review]
Awesome!!, we are now good to go

**Comment by PhillipMwaniki** [comment]
> Am not a fan of experimental features as much but lets first hear what others have to say about it.

I am for ES8

**Pull request merged**